airflow/dags/libs/news_article/crawler/contentCrawling.py

```
@@ -0,0 +1,115 @@
+from bs4 import BeautifulSoup
+import requests
+import time
+import pandas as pd
+import numpy as np
+import re
+import random
+
+def get_content(url):
+    r = requests.get(url)
```

Ruff (0.12.2)

Probable use of `requests` call without timeout

(S113)